ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TopicInfoCache.java

```diff
@@ -141,6 +152,74 @@ private TopicInfo load(final String topicName) {
     }
   }
 
+  private static LogicalSchema getInternalSchema(
+      final String topicName,
+      final PersistentQueryMetadata query
+  ) {
+    if (LEFT_INTERNAL.matcher(topicName).matches()) {
+      return getSchemaFromQuery("PROJECT", "PrependAliasLeft", query);
```

Comment:
this sure ain't pretty, but it might make it a little nicer if we at least extracted constants for the PlaneNodeIds that we're referencing here?